corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -236,7 +236,7 @@ def test_deprecated_form_migration(self):
             update={
                 'property': 'original value'
             }
-        ).as_string()
+        ).as_string().decode('utf-8')
```

Comment:
Would it be hard to deprecate `as_string()` and add `as_text()` and possibly `as_bytes()` methods to `CaseBlock`?

Reply:
Updated: d60a9bdba2024efe9473f0005b37c96cc248e1f0 and 0c4abe2fd0999f8a31e077925e83be85e95e8b28